src/pipeline/build.ts

```diff
@@ -26,6 +26,8 @@ export async function build(args: {
     project: VirtualFileSystem;
     stdlib: string | VirtualFileSystem;
     logger?: TactLogger | null | undefined;
+    checkOnly?: boolean;
+    func?: boolean;
```

Comment:
I'd prefer we modify `ConfigProject` so `checkOnly` and `func` become `config`'s fields. Otherwise the `build`'s function parameter list will be to large with time

Reply:
Will do.